style.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+
+/* initial styling */
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  background-color: #191765;
+  color: white;
+}
```

Comment:
Some of your items are not using the right `font-family` according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346). That includes the logo, description text, and skills list.

[OPTIONAL] Please consider applying the `Roboto` font family in the body selector so that it is applied to the entire page as in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346): 
![image](https://user-images.githubusercontent.com/38088818/164699325-14c52d84-7dce-49f9-a3d3-0acd12ebf97f.png)